charts/tfy-prometheus-config/templates/podMonitor/k8s-pods.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: kubernetes-pods-podmonitor
+  namespace: prometheus
+spec:
+  namespaceSelector:
+    any: true
+  selector:
+    matchExpressions:
+    - key: prometheus.io/scrape
+      operator: In
+      values:
+      - "true"
+  podMetricsEndpoints:
+  - relabelings:
+    - sourceLabels: [ __meta_kubernetes_namespace ]
+      regex: '(istio-system|cert-manager|kube-system)'
+      action: drop
+    - sourceLabels: [ __meta_kubernetes_pod_container_init ]
+      regex: 'true'
+      action: drop
```

Comment:
Let's remove this? or have a option to select if we want to keep it drop it

For istio native sidecars mode we want to scrape init containers too
Maybe if possible we can have condition to only scrape when name of the init container is `istio-proxy`